perma_web/perma/wsgi.py

```diff
@@ -9,36 +9,36 @@
 """
 import os
 
-import newrelic
-from werkzeug.wsgi import DispatcherMiddleware
-
-newrelic.agent.initialize('/srv/www/perma/services/newrelic/newrelic.ini')
+# Newrelic setup
+use_newrelic = os.environ.get("USE_NEW_RELIC", False)
```

Comment:
Instead of using a custom environment variable, your should use the standard NEW_RELIC_ENVIRONMENT variable  to set the environment to one of the environments specified at the bottom of the file and then use monitor_mode=true/false to control collection.